app/javascript/components/UI/Menu.js

```diff
@@ -97,12 +97,13 @@ const Menu = ({ className = '', data, steps, draft, handleSaveDraft, isResult =
   };
 
   return (
+    // TODO className="placement-menu"
```

Comment:
It's not clear to me what should be done by this TODO statement

Reply:
I removed unnecessary TODO-comment